query_frontend/src/parser.rs

```diff
@@ -332,8 +332,14 @@ impl<'a> Parser<'a> {
         let table_name = self.parser.parse_object_name()?.into();
         let (columns, constraints) = self.parse_columns()?;
 
-        // PARTITION BY...
-        let partition = self.maybe_parse_and_check_partition(Keyword::PARTITION, &columns)?;
+        // Parse the simple partition rule, starting with `Partitions $Num`.
```

Comment:
Why not make it `partition random partitions xxx`?
It seems a bit strange, and I think it will mislead people that the `random partitioning` is default partitioning way in ceresdb?

Reply:
Sounds reasonable.